Assets/Script/Player/PlayerContainer.cs

```diff
@@ -312,5 +317,65 @@ public static bool HasAnyBotsActive()
         {
             return _players.Exists(i => i.Profile.IsBot);
         }
+
+        public static void MoveUp(YargPlayer player)
+        {
+            int index = _players.IndexOf(player);
+            if (index == -1)
+            {
+                throw new ArgumentException("Player not found in the active player list");
+            }
+
+            if (index == 0)
+            {
+                return;
+            }
+            _players.RemoveAt(index);
+            _players.Insert(index - 1, player);
+        }
+
+        public static void MoveDown(YargPlayer player)
+        {
+            int index = _players.IndexOf(player);
+            if (index == -1)
+            {
+                throw new ArgumentException("Player not found in the active player list");
+            }
+            if (index == _players.Count - 1)
+            {
+                return;
+            }
+            _players.RemoveAt(index);
+            _players.Insert(index + 1, player);
+        }
+
+        public static void SaveProfileOrder()
+        {
+            foreach (var profile in Profiles)
+            {
+                profile.AutoConnectOrder = null;
+            }
+
+            for (int x = 0; x < _players.Count; x++)
+            {
+                _players[x].Profile.AutoConnectOrder = x;
+            }
+        }
+
+        public static void AutoConnectProfiles()
+        {
+            foreach (var profile in Profiles.Where(e => e.AutoConnectOrder != null).OrderBy(e => e.AutoConnectOrder))
+            {
+                var result = CreatePlayerFromProfile(profile, true);
```

Comment:
The result variable is not used here so it can just be removed:
```suggestion
                CreatePlayerFromProfile(profile, true);
```